### DISCUSSION TITLE
Easy way to make llama_cpp.server available on local net?

### DISCUSSION THREAD

**Discussion by Free-Radical** [abetlen/llama-cpp-python#353] [Q&A]
Hi, I am aware I can clone the repo and modify `llama_cpp/server/__main__.py` to change `localhost` to `0.0.0.0 `, etc but as I would prefer not to do that and am using `pip install llama-cpp-python` to stay up to date with the package:

What is the easiest way to make the **llama-cpp** API server available on the local net with out breaking the `pip install` method?

**Comment by sebaxzero** [comment]
see #315 
if you use windows:

set HOST='external IP of your OS'
python -m llama_cpp.server

**Comment by Free-Radical** [comment]
Great, Thanks!
